packages/datatrak-web/src/views/AccountSettingsPage/PersonalDetailsSection/PersonalDetailsForm.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { Form, FormInput, TextField } from '@tupaia/ui-components';
+import { Button } from '../../../components';
+import { UserAccountDetails } from '../../../types';
+import { successToast } from '../../../utils';
+import { useCurrentUser, useEditUser } from '../../../api';
+
+type PersonalDetailsFormFields = Pick<
+  UserAccountDetails,
+  'firstName' | 'lastName' | 'employer' | 'position' | 'mobileNumber'
+>;
+
+const ButtonWrapper = styled.div`
+  grid-column: -2;
+`;
+
+const StyledTextField = styled(TextField)`
+  margin: 0; // Use gap on parent to control spacing
+`;
+
+const StyledForm = styled(Form)`
+  max-width: 44.25rem;
+  width: 100%;
+`;
+
+const StyledFieldset = styled.fieldset`
+  border: none;
+  margin: 0;
+  padding: 0;
```

Comment:
Really, these should be in a reset stylesheet. If we have one that isn’t just the one that comes with Material UI, do let me know and I can move this!